drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -850,6 +850,26 @@ void AudioDriverWASAPI::thread_func(void *p_udata) {
 								r = read_sample(ad->audio_input.format_tag, ad->audio_input.bits_per_sample, data, j * 2 + 1);
 							} else if (ad->audio_input.channels == 1) {
 								l = r = read_sample(ad->audio_input.format_tag, ad->audio_input.bits_per_sample, data, j);
+							} else if (ad->audio_input.channels >= 2) {
```

Comment:
`== 2` is already handled above so this should be `> 2`.

Reply:
Oh! Haha🤣. Thank you for your modifications.